manifests/cinder-csi-plugin/cinder-csi-controllerplugin.yaml

```diff
@@ -59,7 +59,7 @@ spec:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
         - name: csi-snapshotter
-          image: quay.io/k8scsi/csi-snapshotter:v1.2.2
+          image: quay.io/k8scsi/csi-snapshotter:v2.0.1
```

Comment:
If we are updating to 2.0.1 , I believe we need snapshot-controller also to be deployed?

Reply:
the reason I made this PR is because I found some metrics can only be showed in latest
e.g 2.0.0, but I am not aware the snapshot-controller need to be there
so some homework need to be done.. thanks for pointing that

Reply:
ok, I found 2.0.0 requires 1.17 k8s and controller to be deployed
so it might takes time for us to upgrade to there

anyway, will open an issue on that

Reply:
@jichenjc the metrics is part of which sidecar? provisioner?

Reply:
@ramineni the metrics is introduced at 2.0.0, so because 2.0.0 has the controller 
requirement, then we have to first have the controller first

Reply:
@jichenjc ok, please update the description as well as now this PR is not updating snapshot sidecar, metrics is not applicable still , right.